### PR TITLE
export f000 grib2 file and upload to s3

### PR DESCRIPTION
**Pull request by SadeghTabas-NOAA**
### Description
This PR exports the f000 grib2 file and uploads it to the s3 bucket. 

### Linked Issues
- Closes https://github.com/NOAA-EMC/graphcast/issues/51

### Input data
- [x] No changes are expected to input data.
- [ ] Changes are expected to input data:
  - [ ] New input data.
  - [ ] Updated input data.

### Needed libraries

-

### CODE REVIEW

**Comment by SadeghTabas-NOAA** [comment]
Sample F000 grib2 file content:

[Sadegh.Tabas@gcpmlsadegh-8 /]$ wgrib2 /contrib/Sadegh.Tabas/test/check1/graphcastgfs.t00z.pgrb2.0p25.f000
1:0:d=2024061500:UGRD:10 m above ground:anl:
2:3244679:d=2024061500:VGRD:10 m above ground:anl:
3:6489358:d=2024061500:TMP:2 m above ground:anl:
4:9734037:d=2024061500:HGT:50 mb:anl:
5:12978716:d=2024061500:HGT:100 mb:anl:
6:16223395:d=2024061500:HGT:150 mb:anl:
7:19468074:d=2024061500:HGT:200 mb:anl:
8:22712753:d=2024061500:HGT:250 mb:anl:
9:25957432:d=2024061500:HGT:300 mb:anl:
10:29202111:d=2024061500:HGT:400 mb:anl:
11:32446790:d=2024061500:HGT:500 mb:anl:
12:35691469:d=2024061500:HGT:600 mb:anl:
13:38936148:d=2024061500:HGT:700 mb:anl:
14:42180827:d=2024061500:HGT:850 mb:anl:
15:45425506:d=2024061500:HGT:925 mb:anl:
16:48670185:d=2024061500:HGT:1000 mb:anl:
17:51914864:d=2024061500:PRMSL:mean sea level:anl:
18:55159543:d=2024061500:SPFH:50 mb:anl:
19:58404222:d=2024061500:SPFH:100 mb:anl:
20:61648901:d=2024061500:SPFH:150 mb:anl:
21:64893580:d=2024061500:SPFH:200 mb:anl:
22:68138259:d=2024061500:SPFH:250 mb:anl:
23:71382938:d=2024061500:SPFH:300 mb:anl:
24:74627617:d=2024061500:SPFH:400 mb:anl:
25:77872296:d=2024061500:SPFH:500 mb:anl:
26:81116975:d=2024061500:SPFH:600 mb:anl:
27:84361654:d=2024061500:SPFH:700 mb:anl:
28:87606333:d=2024061500:SPFH:850 mb:anl:
29:90851012:d=2024061500:SPFH:925 mb:anl:
30:94095691:d=2024061500:SPFH:1000 mb:anl:
31:97340370:d=2024061500:TMP:50 mb:anl:
32:100585049:d=2024061500:TMP:100 mb:anl:
33:103829728:d=2024061500:TMP:150 mb:anl:
34:107074407:d=2024061500:TMP:200 mb:anl:
35:110319086:d=2024061500:TMP:250 mb:anl:
36:113563765:d=2024061500:TMP:300 mb:anl:
37:116808444:d=2024061500:TMP:400 mb:anl:
38:120053123:d=2024061500:TMP:500 mb:anl:
39:123297802:d=2024061500:TMP:600 mb:anl:
40:126542481:d=2024061500:TMP:700 mb:anl:
41:129787160:d=2024061500:TMP:850 mb:anl:
42:133031839:d=2024061500:TMP:925 mb:anl:
43:136276518:d=2024061500:TMP:1000 mb:anl:
44:139521197:d=2024061500:UGRD:50 mb:anl:
45:142765876:d=2024061500:UGRD:100 mb:anl:
46:146010555:d=2024061500:UGRD:150 mb:anl:
47:149255234:d=2024061500:UGRD:200 mb:anl:
48:152499913:d=2024061500:UGRD:250 mb:anl:
49:155744592:d=2024061500:UGRD:300 mb:anl:
50:158989271:d=2024061500:UGRD:400 mb:anl:
51:162233950:d=2024061500:UGRD:500 mb:anl:
52:165478629:d=2024061500:UGRD:600 mb:anl:
53:168723308:d=2024061500:UGRD:700 mb:anl:
54:171967987:d=2024061500:UGRD:850 mb:anl:
55:175212666:d=2024061500:UGRD:925 mb:anl:
56:178457345:d=2024061500:UGRD:1000 mb:anl:
57:181702024:d=2024061500:VGRD:50 mb:anl:
58:184946703:d=2024061500:VGRD:100 mb:anl:
59:188191382:d=2024061500:VGRD:150 mb:anl:
60:191436061:d=2024061500:VGRD:200 mb:anl:
61:194680740:d=2024061500:VGRD:250 mb:anl:
62:197925419:d=2024061500:VGRD:300 mb:anl:
63:201170098:d=2024061500:VGRD:400 mb:anl:
64:204414777:d=2024061500:VGRD:500 mb:anl:
65:207659456:d=2024061500:VGRD:600 mb:anl:
66:210904135:d=2024061500:VGRD:700 mb:anl:
67:214148814:d=2024061500:VGRD:850 mb:anl:
68:217393493:d=2024061500:VGRD:925 mb:anl:
69:220638172:d=2024061500:VGRD:1000 mb:anl:
70:223882851:d=2024061500:VVEL:50 mb:anl:
71:227127530:d=2024061500:VVEL:100 mb:anl:
72:230372209:d=2024061500:VVEL:150 mb:anl:
73:233616888:d=2024061500:VVEL:200 mb:anl:
74:236861567:d=2024061500:VVEL:250 mb:anl:
75:240106246:d=2024061500:VVEL:300 mb:anl:
76:243350925:d=2024061500:VVEL:400 mb:anl:
77:246595604:d=2024061500:VVEL:500 mb:anl:
78:249840283:d=2024061500:VVEL:600 mb:anl:
79:253084962:d=2024061500:VVEL:700 mb:anl:
80:256329641:d=2024061500:VVEL:850 mb:anl:
81:259574320:d=2024061500:VVEL:925 mb:anl:
82:262818999:d=2024061500:VVEL:1000 mb:anl:

File: NCEP/run_graphcast.py [hunk 1]
```diff
@@ -179,9 +180,20 @@ def get_predictions(self):
         self.save_grib2(forecasts)
 
     def save_grib2(self, forecasts):
+        converter = Netcdf2Grib()
+
+        # Call and save f000 in grib2
+        ds = self.current_batch
+        ds = ds.drop_vars(['geopotential_at_surface','land_sea_mask', 'total_precipitation_6hr'])
```
Comment (junwang-noaa) [inline NCEP/run_graphcast.py:187]: Can we only drop total_precipitation_6hr and keep hgtsfc and land sea mask?

Comment (SadeghTabas-NOAA) [reply NCEP/run_graphcast.py:187]: Those are not in the forecasts grib2 file, do you still need them in the f000 file?

Comment (junwang-noaa) [reply NCEP/run_graphcast.py:187]: I see. Let's drop them then to be consistent with other forecasts.

**Pull request merged**